golem-worker-service/Cargo.toml

```diff
@@ -15,48 +15,48 @@ path = "src/main.rs"
 [lib]
 
 [dependencies]
-golem-common = { path = "../golem-common" }
 golem-api-grpc = { path = "../golem-api-grpc" }
+golem-common = { path = "../golem-common" }
 golem-service-base = { path = "../golem-service-base" }
-golem-worker-service-base = {path = "../golem-worker-service-base"}
+golem-wasm-ast = { workspace = true }
 golem-wasm-rpc = { workspace = true }
+golem-worker-service-base = { path = "../golem-worker-service-base" }
 
-golem-wasm-ast = { workspace = true }
+anyhow = { workspace = true }
 async-trait = { workspace = true }
-regex = "1.10.3"
-bincode = "2.0.0-rc.3"
-serde = { version = "1.0.196", features = ["derive"] }
-serde_json = "1.0.113"
-serde_yaml = "0.9.31"
-strum = "0.26.1"
-strum_macros = "0.26.1"
-poem = { workspace = true, features = ["prometheus", "opentelemetry", "test"] }
-poem-openapi = { workspace = true }
-derive_more = "0.99.17"
-uuid = { workspace = true }
-lazy_static = { workspace = true }
-nom = "7.1.3"
-tracing = { workspace = true }
-tracing-subscriber = { workspace = true, features = ["fmt", "std", "json"] }
-http = "1.0.0"
-humantime-serde = { workspace = true }
-tonic = "0.10.2"
-tonic-health = { workspace = true }
-tonic-reflection = { workspace = true }
+bincode = { workspace = true }
+bytes = { workspace = true }
+derive_more = { workspace = true }
 figment = { workspace = true }
-url = { workspace = true }
-http_02 = { package = "http", version = "0.2.11" }
+futures = { workspace = true }
+futures-util = { workspace = true }
+http = { workspace = true }
+http_02 = { workspace = true }
+humantime-serde = { workspace = true }
 hyper = { workspace = true }
-bytes = { workspace = true }
-tokio = { workspace = true }
-tokio-stream = { workspace = true }
-tokio-util = "0.7.10"
-prometheus = "0.13.3"
-futures = "0.3.30"
-openapiv3 = "2.0.0"
+lazy_static = { workspace = true }
+nom = { workspace = true }
+openapiv3 = { workspace = true }
 opentelemetry = { workspace = true }
-opentelemetry_sdk = { workspace = true }
 opentelemetry-prometheus = { workspace = true }
-futures-util = { workspace = true }
-tap = "1.0.1"
-anyhow.workspace = true
+opentelemetry_sdk = { workspace = true }
+poem = { workspace = true }
+poem-openapi = { workspace = true }
+prometheus = { workspace = true }
+regex = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+serde_yaml = { workspace = true }
+strum = { workspace = true }
+strum_macros = { workspace = true }
+tap = { workspace = true }
+tokio = { workspace = true }
+tokio-stream = { workspace = true }
+tokio-util = { workspace = true }
+tonic = { workspace = true }
+tonic-health = { workspace = true }
+tonic-reflection = { workspace = true }
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
+url = { workspace = true }
+uuid = { workspace = true }
```

Comment:
Does these changes made it faster as well? As in making sure workspace=true is applied?